packages/components/src/slot-fill/test/__snapshots__/slot.js.snap

```diff
@@ -0,0 +1,74 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Slot should re-render Slot when not bubbling virtually 1`] = `
+Array [
+  <div>
+    1
+  </div>,
+  <button
+    onClick={[Function]}
+    type="button"
+  />,
+]
+`;
+
+exports[`Slot should re-render Slot when not bubbling virtually 2`] = `
+Array [
+  <div>
+    2
+  </div>,
+  <button
+    onClick={[Function]}
+    type="button"
+  />,
+]
+`;
+
+exports[`Slot should render a Fill containing an array 1`] = `
+<div>
+  <span />
+  <div />
+  text
+</div>
+`;
+
+exports[`Slot should render a Fill containing an element 1`] = `
+<div>
+  <span />
+</div>
+`;
+
+exports[`Slot should render a string Fill 1`] = `
+<div>
+  content
+</div>
+`;
+
+exports[`Slot should render a string Fill with HTML wrapper when render props used 1`] = `
+<div>
+  <blockquote>
+    content
+  </blockquote>
+</div>
+`;
+
+exports[`Slot should render empty Fills 1`] = `<div />`;
+
+exports[`Slot should render empty Fills without HTML wrapper when render props used 1`] = `<div />`;
+
+exports[`Slot should render in expected order 1`] = `
+Array [
+  <div>
+    first
+    second
+  </div>,
+  <button
```

Comment:
Note that, previously, we weren't doing snapshot testing and the `button` component wasn't checked for in this test.